velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -319,11 +328,39 @@ void AggregationFuzzer::go() {
 
       std::vector<TypePtr> types;
       std::vector<std::string> names;
-
       auto groupingKeys = generateKeys("g", names, types);
+
+      std::vector<TypePtr> payloadTypes;
+      std::vector<std::string> payloadNames;
+      generateTypes("c", 1, payloadNames, payloadTypes);
```

Comment:
why not pass names and types to generateTypes directly? Looks like generateTypes appends names and types to provides lists already.